src/chrome/content/rules/FoodRepublic.com.xml

```diff
@@ -0,0 +1,12 @@
+<!--
+	Active mixed content:
```

Comment:
I only found mixed fonts and trackers so far.

Reply:
Fixed, thanks!